### PR TITLE
Corrected minor typos

### PR DESCRIPTION
**Pull request by Marie-Donnie**
Plus les blancs à la fin des lignes retirés par emacs.

### CODE REVIEW

File: slides/ocl.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Sometimes, the UML lacks precision. Suppose the following class diagram:
 
 
 - How do you specify that this class only considers people born after 1900?
-- And how do you specify that cycles are not allowed (i.e., that a person cannot be an ancestral of himself)?
```
Comment (sunye) [inline slides/ocl.md:51]: "ancestor". "Ancestral" is an adjective.

**Review by sunye — CHANGES_REQUESTED** [review]
Thanks for the corrections. However, I don't agree with the first change.

File: slides/ocl.md (hunk 1 above)
Comment (Marie-Donnie) [reply slides/ocl.md:51]: Ancestor was the word I was suggesting, it was ancestral before.

**Review by sunye — APPROVED** [review]

File: slides/ocl.md (hunk 1 above)
Comment (sunye) [reply slides/ocl.md:51]: My bad, sorry for this.

**Pull request merged**